.travis.yml

```diff
@@ -20,10 +20,8 @@ env:
     - REPO=dhlabbasel/sipi
 
 script:
-  # decrypt archive containing the kakadu library
-  - openssl aes-256-cbc -K $encrypted_7deda1f68196_key -iv $encrypted_7deda1f68196_iv -in private/private.tar.enc -out private/private.tar -d
-  - tar xvf private/private.tar
-  - mv v7_9-01727L.zip vendor/.
+  # decrypt the kakadu library
+  - openssl aes-256-cbc -K $encrypted_53935862068b_key -iv $encrypted_53935862068b_iv -in vendor/v7_9-01727L.zip.enc -out vendor/v7_9-01727L.zip -d
```

Comment:
Isn't this the old version that is going to be replaced by Lukas's PR?

Reply:
Or will that be replaced once the PR is merged?

Reply:
This is the current (old) version used by `develop` and which will be replaced by the open PR from Lukas. Just make sure to remove it in the PR (or I could do it).

Reply:
Could you do it, please?

Reply:
jup :-)